com.unity.render-pipelines.high-definition/Documentation~/Dynamic-Resolution.md

```diff
@@ -18,16 +22,18 @@ To use dynamic resolution in your Project, you must enable dynamic resolution in
 1. In the Inspector for your HDRP Asset, go to **Rendering** **> Dynamic Resolution** and enable the **Enable** checkbox. For information on how to customize the rest of the HDRP Asset’s global dynamic resolution properties, see the dynamic resolution section of the [HDRP Asset documentation](HDRP-Asset.md#DynamicResolution).
 2. For every [Camera](HDRP-Camera.md) you want to perform dynamic resolution, go to the **General** section and enable **Allow Dynamic Resolution**.
 
-Dynamic resolution is not automatic, so you need to manually call the `DynamicResolutionHandler.SetDynamicResScaler(PerformDynamicRes scaler, DynamicResScalePolicyType scalerType)` function.
+HDRP does not use dynamic resolution automatically. To use dynamic resolution in your scene, call the following function:
 
-The policy type can be one of the following:
+```DynamicResolutionHandler.SetDynamicResScaler(PerformDynamicRes scaler, DynamicResScalePolicyType scalerType)```
+
+This function can use one of the following policy types:
 
 - `DynamicResScalePolicyType.ReturnsPercentage`:  The DynamicResolutionHandler expects the `scaler` to return a screen percentage value. The value set will be clamped between the minimum and maximum percentage set in the HDRP Asset.
 - `DynamicResScalePolicyType.ReturnsMinMaxLerpFactor`:  The DynamicResolutionHandler expects the `scaler` to return a factor `t` that is the [0 ... 1] range and that will be used as a lerp factor between the minimum and maximum screen percentages set in the HDRP asset.
 
-It is best practice to keep the screen percentage above 50%. However, HDRP supports values from 5% to 100%, anything below or above this range gets clamped accordingly.
+HDRP supports values from 5% to 100% and clamps anything above 100%. It is best practice to keep the screen percentage above 50%.
 
-The example below shows how to call this function. In a real production environment, you would call this function depending on the performance of your application.
+The example below demonstrates how to call this function. During development, call this function depending on the performance of your application.
```

Comment:
This change kinda loses the purpose of the original one that was meant to say "This is just a dumb example, please do something more proper"

Reply:
Ah I see :D Thanks!